CONTRIBUTING.rst

```diff
@@ -137,15 +149,20 @@ To set up ScanCode for local development:
        cd <resulting libbz2.so directory>
        sudo ln -s <your version of libbz2.so> libbz2.so.1.0
       
-   See https://github.com/nexB/scancode-toolkit/issues/443 for more information.
+   See `this issue <https://github.com/nexB/scancode-toolkit/issues/443>`_ for more information.
 
-7. Commit your changes and push your branch to your GitHub fork::
+7. Check the status of your local repository before commit, regarding files changed::
+    
+    git status
 
-    git add .
+
+8. Commit your changes and push your branch to your GitHub fork::
+
+    git add <file-changed-1> <file-changed-2> <file-changed-3> / git add .
```

Comment:
I like how this discourages `git add .`. But the way it is written now can be misunderstood as having to type the whole line, including ` / git add .`. Should we remove that completely?

Reply:
Consider you are working on an issue which is likely to get solved by 3-4 commits , in this case git add should consider only those file which are about to commit right now.

Reply:
I think we are in agreement here that `git add .` should be avoided. I was just wondering whether we should remove it completely from this line.

Reply:
correct